### ISSUE TITLE
Request:   PMP native stand-among user-installable app for Linux

### ISSUE THREAD

**Issue by ChuckPa** [plexinc/plex-media-player#612]
As stated in title.   Stand among app, installable as a package just like any other.

For details,  please see this thread where one of our users has already demonstrated it working and is using AppImage.   

If he can spin the build,  and knowing libvaapi version 2.0 will soon be available which will resolve HW acceleration issues for all the capable Intel processors,   why can't we? 

http://forums.plex.tv/discussion/278570/plex-media-player-packages-for-linux

**Comment by wm4** [comment]
At this point it's mostly about impending build infrastructure changes, which are work in progress. So, on hold. Once that is done, we can think about this. I think.

**Comment by LongChair** [comment]
@markus101 : is that something we should consider ?

**Comment by mseeley** [comment]
No plans for dedicated Linux builds in the near future.
